sequencing/ATACSeq/jobSubmission/1_batchRunPreAnalysis.sh

```diff
@@ -0,0 +1,159 @@
+## PROOFREADER : Marina Flores-Payan                                                                                  ||
+## CONTACT: m.flores-payan@exeter.ac.uk                                                                               ||
+## LAST UPDATE: February 2024                                                                                         ||
+## ===================================================================================================================##
+## EXECUTION: sbatch --array= ./sequencing/ATACSeq/jobSubmission/1_batchRunPreAnalysis.sh <projectName> <option>      ||
+## - execute from scripts directory                                                                                   ||
+##                                                                                                                    ||
+## INPUTS:                                                                                                            || 
+## --array -> Number of jobs to run. Will select sample(s) corresponding to the number(s) input                       ||
+## $1 -> <projectName> Name of project                                                                                ||
+## $2 -> <option> Specify step to run: FASTQC, TRIM, ALIGN or ENCODE. Can be combined. Default is to                  ||
+## run all                                                                                                            ||
+##                                                                                                                    ||
+## DESCRIPTION: This script performs the first step in the ATAC-seq pipeline: pre-analysis. This                      ||
+## includes quality control of raw reads (FASTQC), trimming the reads (TRIM), aligning reads to                       ||
+## reference genome (ALIGN) and calculate ENCODEQC metrics (ENCODE).                                                  ||
+##                                                                                                                    ||
+## REQUIRES:                                                                                                          ||
+## - File in ${METADIR}/samples.txt that lists sample names.                                                          ||
+## ===================================================================================================================##
+
+## ============ ##
+##    SET UP    ##
+## ============ ##
+
+## print start date and time when script runs
+echo Job started on:
+date -u
+
+## needs to be executed from the scripts folder
+echo "Changing Folder to: " $SLURM_SUBMIT_DIR
+cd $SLURM_SUBMIT_DIR || exit 1
+
+## load config file provided on command line related to the specified project
+echo "Loading config file for project: " $1
+export PROJECT=$1
+source ./ATACSeq/config/config.txt 
+
```

Comment:
If I remember correctly, $PROJECT is used in the config so that the config file can be more modular and work with multiple projects.
In a later pull request (doesn't need to be this one), it would be nice if the config file is one of the inputs from the user instead of $PROJECT. Putting the config file as a user input will also mean that the script doesn't need to be executed from the root of the repository.
This would require the config file to be specific to a project (though this is what was pushed for a couple of weeks ago), and $PROJECT wouldn't be required anymore.